src/main/java/week2Task2/Men.java

```diff
@@ -0,0 +1,39 @@
+package week2Task2;
+
+public class Men extends Human {
+    private String beardColor;
+    private int bicepsVolume;
+    @Override
+    void behaviourAndEssence(){
+        System.out.println("Behaviour is: " + "Bad man" + "\n" + "Name is: " + getName() + "\n" + "Age is: " + getAge() + "\n" +
+               "Height is: " + getHeight() + "\n" + "Weight is: " +  getWeight() + "\n" + "Hair color is: " + getHairColor() + "\n" +
+                "Beard color is " + getBeardColor() + "\n" + "Biceps volume is: " + getBicepsVolume());
+    }
```

Comment:
overriding makes sense when you want to override behaviour from the parent class, currently, in paret class you don't have any realisation

Reply:
removed and overwritten in methods showMan and showWoman